src/main/core/Resources/config/services/subscriber.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    _defaults:
+        autowire: true
```

Comment:
Better not use autowiring on this project, to avoid compile-time perf issues due to the big amount of services and circular dependencies it has currently.